v3/src/components/graph/adornments/count/count-model.ts

```diff
@@ -1,12 +1,43 @@
-import { Instance } from "@concord-consortium/mobx-state-tree"
+import { Instance, types } from "mobx-state-tree"
 import { AdornmentModel, IAdornmentModel } from "../adornment-models"
 import { kCountType } from "./count-types"
 
 export const CountModel = AdornmentModel
   .named('CountModel')
   .props({
-    type: 'Count'
+    type: 'Count',
+    showCount: false,
+    showPercent: false,
+    percentType: types.optional(types.enumeration(["cell", "column", "row"]), "row"),
   })
+  .views(self => ({
+    percentValue(dataConfig: any, casesInPlot: number, cellKey: Record<string, string>) {
+      const casesInRow = dataConfig?.rowCases(cellKey)?.length ?? 0
+      const casesInColumn = dataConfig?.columnCases(cellKey)?.length ?? 0
+      const casesInCell = dataConfig?.cellCases(cellKey)?.length ?? 0
+      const divisor = self.percentType === "row"
+        ? casesInRow
+        : self.percentType === "column"
+          ? casesInColumn
+          : casesInCell
```

Comment:
This is going to compute all three even though we only need one. How about:
```suggestion
      const divisor = self.percentType === "row"
        ? dataConfig?.rowCases(cellKey)?.length ?? 0
        : self.percentType === "column"
          ? dataConfig?.columnCases(cellKey)?.length ?? 0
          : dataConfig?.cellCases(cellKey)?.length ?? 0
```